pkg/channel/distributed/dispatcher/dispatcher/handler_test.go

```diff
@@ -150,13 +161,26 @@ func TestHandlerConsumeClaim(t *testing.T) {
 	// Execute The Individual Test Cases
 	for _, testCase := range filteredTestCases {
 		t.Run(testCase.name, func(t *testing.T) {
-			performHandlerConsumeClaimTest(t, testCase.destinationUri, testCase.replyUri, testCase.deadLetterUri, testCase.retry)
+			performHandlerConsumeClaimTest(t,
+				testCase.destinationUri,
+				testCase.replyUri,
+				testCase.deadLetterUri,
+				testCase.retry,
+				testCase.dispatchErr,
+				testCase.expectSkipMark,
+			)
 		})
```

Comment:
maybe it's time we just pass the testCase to the performHandlerConsumeClaimTest() function ?

Reply:
sounds good